### PR TITLE
[MFTF] add new  AdminOpenConfigurationStoresPageActionGroup

### LINKED ISSUE #29848: [Issue] [MFTF] add new  AdminOpenConfigurationStoresPageActionGroup
This issue is automatically created based on existing pull request: magento/magento2#29827: [MFTF] add new  AdminOpenConfigurationStoresPageActionGroup

---------
This PR add new `AdminOpenConfigurationStoresPageActionGroup`  and use this one instead 
`<amOnPage url="{{ConfigurationStoresPage.url}}" stepKey="navigateToWYSIWYGConfigPage1"/>`
       ` <waitForPageLoad stepKey="wait1"/>`

### PR DESCRIPTION
**Pull request by Usik2203**
This PR add new `AdminOpenConfigurationStoresPageActionGroup`  and use this one instead 
`<amOnPage url="{{ConfigurationStoresPage.url}}" stepKey="navigateToWYSIWYGConfigPage1"/>`
       ` <waitForPageLoad stepKey="wait1"/>`

### Resolved issues:
1. [x] resolves magento/magento2#29848: [MFTF] add new  AdminOpenConfigurationStoresPageActionGroup

### CODE REVIEW

**Comment by Usik2203** [comment]
@magento run Functional Tests CE , Functional Tests EE , Functional Tests B2B

File: app/code/Magento/Backend/Test/Mftf/ActionGroup/AdminOpenConfigurationStoresPageActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  */
+-->
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminOpenConfigurationStoresPageActionGroup">
+        <annotations>
+            <description>Open configuration stores page.</description>
+        </annotations>
+
+        <amOnPage url="{{ConfigurationStoresPage.url}}" stepKey="goToConfigurationStoresPage"/>
```
Comment (lbajsarowicz) [inline app/code/Magento/Backend/Test/Mftf/ActionGroup/AdminOpenConfigurationStoresPageActionGroup.xml]: Could you also deprecate the old `Page` object and introduce new one that actually follows the Magento Best Practices (`AdminConfigurationStoresPage`)?

**Review by lbajsarowicz — CHANGES_REQUESTED** [review]
The Page does not follow Magento Best Practices.
Are you able to adjust that in scope of this PR?

**Comment by sidolov** [comment]
@magento create issue

**Comment by lbajsarowicz** [comment]
@magento run Functional Tests CE, Functional Tests EE, Functional Tests B2B

**Comment by lbajsarowicz** [comment]
@magento run Functional Tests CE, Functional Tests EE, Functional Tests B2B

Failing tests **are not related** with the scope of change.

**Comment by Usik2203** [comment]
@magento run Functional Tests CE, Functional Tests EE, Functional Tests B2B

**Comment by Usik2203** [comment]
@magento run Functional Tests CE, Functional Tests EE, Functional Tests B2B

File: app/code/Magento/Backend/Test/Mftf/ActionGroup/AdminOpenConfigurationStoresPageActionGroup.xml (hunk 1 above)
Comment (Usik2203) [reply app/code/Magento/Backend/Test/Mftf/ActionGroup/AdminOpenConfigurationStoresPageActionGroup.xml]: @lbajsarowicz  Done

**Comment by gabrieldagama** [comment]
The risk was set to`low` due to: the suggested changes shouldn't affect other areas.

**Comment by Usik2203** [comment]
Hi @lbajsarowicz  
Could i kindly ask you to check this PR ?
Thanks

**Review by sidolov — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sidolov, thank you for the review.
[ENGCOM-8259](https://jira.corp.magento.com/browse/ENGCOM-8259) has been created to process this Pull Request
:eight_spoked_asterisk: @sidolov, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by gabrieldagama** [comment]
@magento give me new instance with extensions https://github.com/magento-commerce/aep-launch

**Comment by gabrieldagama** [comment]
@magento give me new instance with extensions magento/magento2-jp

**Pull request merged**